### PR TITLE
fixed bug

### PR DESCRIPTION
**Pull request by natischmidt**
fixed bug, now u can add new host while leaving the blocked users empty if u dont need to add it.

### CODE REVIEW

File: noq_django/backend/migrations/0002_roomtype_alter_available_table_alter_booking_table_and_more.py
Comment (jani-at-noq) [inline noq_django/backend/migrations/0002_roomtype_alter_available_table_alter_booking_table_and_more.py:1]: This should not be part of PR

File: noq_django/backend/migrations/0003_remove_booking_room_alter_available_table_and_more.py
Comment (jani-at-noq) [inline noq_django/backend/migrations/0003_remove_booking_room_alter_available_table_and_more.py:1]: This should not be part of PR

File: noq_django/backend/migrations/0004_alter_product_type.py
Comment (jani-at-noq) [inline noq_django/backend/migrations/0004_alter_product_type.py:1]: This should not be part of PR

File: noq_django/backend/migrations/0005_merge_20240714_1246.py
Comment (jani-at-noq) [inline noq_django/backend/migrations/0005_merge_20240714_1246.py:1]: This should not be part of PR

File: noq_django/backend/migrations/0007_merge_20240830_1009.py
Comment (jani-at-noq) [inline noq_django/backend/migrations/0007_merge_20240830_1009.py:1]: This should not be part of PR

File: noq_django/backend/migrations/0008_alter_host_blocked_clients.py
Comment (jani-at-noq) [inline noq_django/backend/migrations/0008_alter_host_blocked_clients.py:1]: This should be 0007

File: noq_django/backend/migrations/0008_alter_host_blocked_clients.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.2.10 on 2024-09-06 10:15
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('backend', '0007_merge_20240830_1009'),
```
Comment (jani-at-noq) [inline noq_django/backend/migrations/0008_alter_host_blocked_clients.py:9]: Dependency should be 0006_invoice_buyer_address_invoice_buyer_name_and_more

File: noq_django/backend/models.py
Comment (jani-at-noq) [inline noq_django/backend/models.py:1]: Looks good!

File: .idea/.gitignore
Comment (jani-at-noq) [inline .idea/.gitignore:1]: I'm a bit against adding Pycharm related files to the repo. Why you want to add them (.idea folder)?

**Review by jani-at-noq — CHANGES_REQUESTED** [review]

**Comment by natischmidt** [comment]
@jani-at-noq just started using vscode, bc my liscense for Intellij ran out.. still getting used to it - wasnt intenionall i'll fix it

**Review by jani-at-noq — APPROVED** [review]
Nu är det bara två filer som det ska vara. Ok för min del.

**Pull request merged**